README.md

```diff
@@ -120,7 +120,9 @@ $ ./chainlink
 
 ```bash
 $ yarn
-$ make contracts
+$ (cd belt && yarn setup)
```

Comment:
Why are these in parenthesis?

Reply:
They preserve the callers context.  You don't have to cd back after.